### PR TITLE
working on st video

### PR DESCRIPTION
**Pull request by FraPochetti**
@gabrielelanaro, I am pushing for reference. Review not needed yet.

### CODE REVIEW

File: prototypes/styletransfer/model.py [hunk 1]
```diff
@@ -191,6 +192,26 @@ def _estimate_content2weight(
...
+        content_weight=1.0,
+        style_weight=1.0,
+        total_variation_weight=1.0
+    ) -> List[np.array]:
+
+        nframes = len(frames)
+        transferred = []
+        for frame in range(0, nframes-1):
+            content_img = frame
+            for result in self.run_style_transfer(style_img=style_img, content_img=content_img, num_iterations=num_iterations):
```
Comment (gabrielelanaro) [inline prototypes/styletransfer/model.py]: maybe we should pass the weights here?

File: prototypes/styletransfer/videos.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        img = Image.open(img_path)
+        img = np.array(img)
+        img = fix_img(img)
+        frames.append(img)
+
+    return frames
+
+def make_video(images: List[np.array], 
+               save_to: str, 
+               video_name: str = "video.avi"):
```
Comment (gabrielelanaro) [inline prototypes/styletransfer/videos.py]: Ok for now since we using it for debugging but may be worth it returning a GIF as well

Comment (FraPochetti) [reply prototypes/styletransfer/videos.py]: Yes, totally. This is just for reference. Needs to be adapted to our use case.

File: prototypes/styletransfer/model.py (hunk 1 above)
Comment (FraPochetti) [reply prototypes/styletransfer/model.py]: @gabrielelanaro let me know if 4ad290c fixes this

Comment (FraPochetti) [reply prototypes/styletransfer/model.py]: I also added the unit test, but still need to actually run it. The test regarding `extract_frames_from_gif` passes.

File: prototypes/styletransfer/videos.py (hunk 2 above)
Comment (gabrielelanaro) [reply prototypes/styletransfer/videos.py]: feel free to merge

**Pull request merged**